tools/project_manager/remote_file_manager.php

```diff
@@ -952,7 +964,8 @@ function get_current_dir_relative_path($home_dirname)
 
     // Only SAs are allowed access to other home folders.
     if (!user_may_access_all_upload_dirs()) {
-        if (!startswith("$abspath/", "$abs_uploads_dir/$home_dirname/")) {
+        if (!startswith("$abspath/", "$abs_uploads_dir/$home_dirname/") &&
+            !startswith("$abspath/", "$abs_uploads_dir/$commons_rel_dir/")) {
             fatal_error( _("You are restricted to your home folder and its descendants") );
```

Comment:
Should this error message be changed to "You are restricted to your home folder or the Commons folder and their descendants"?